boot/bootutil/src/bootutil_misc.c

```diff
@@ -43,6 +44,7 @@ const uint32_t boot_img_magic[] = {
 };
 
 const uint32_t BOOT_MAGIC_SZ = sizeof boot_img_magic;
+const uint32_t BOOT_MAX_ALIGN = MAX_FLASH_ALIGN;
```

Comment:
Rust cannot import `#define`s and a `const` variable with `extern` cannot be used to size arrays in a header so this was the best option IMO.